protected/humhub/components/behaviors/PolymorphicRelation.php

```diff
@@ -169,14 +169,14 @@ private function validateUnderlyingObjectType(?object $object)
     /**
      * Loads an active record based on classname and primary key.
      *
-     * @param string $className
+     * @param string|mixed $className
      * @param string|int $primaryKey
      *
      * @return null|ActiveRecord|ActiveRecordInterface
      */
-    public static function loadActiveRecord(string $className, $primaryKey): ?ActiveRecordInterface
+    public static function loadActiveRecord($className, $primaryKey): ?ActiveRecordInterface
```

Comment:
Would this be also an option? This would allow us to remove the `is_string` check.

```suggestion
    public static function loadActiveRecord(?string $className, $primaryKey): ?ActiveRecordInterface
```

Reply:
We can try. There is no restriction on the `getAttribute()` method. In fact, in some cases, objects are returned. This would lead to a fatal error again. So I wanted to be on the safe side. But happy to apply `?string` ...

Reply:
Thank you, being on the safe side is usually a good thing :-)

But let's risk it here.